aep/general/0151/lro.oas.yaml

```diff
@@ -22,12 +22,15 @@ components:
       properties:
         name:
           type: string
-          description: The server-assigned name, which is only unique within the same service that originally returns it.
```

Comment:
Does this PR need to change this file?

Reply:
nope sorry, don't know what happened there.